ILMerge.Tests/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
-    "NUnit": "3.6.0" 
+    "NUnit": "3.6.0",
+    "NUnit3TestAdapter": "3.6.0"
```

Comment:
My guess is that it would be better to use the [3.7.0](https://github.com/nunit/nunit3-vs-adapter/releases/tag/3.7) test adapter because of the fixes. NUnit is deliberately diverging the versioning and release cycles of the framework, the adapter, and the runners.